api/jobs/start_job_handler.go

```diff
@@ -22,22 +22,43 @@ const (
 )
 
 // HandleStartPipelineJob Handles the creation of a pipeline job for an application
-func (jh JobHandler) HandleStartPipelineJob(appName string, pipeline *pipelineJob.Definition, jobSpec *jobModels.JobParameters) (*jobModels.JobSummary, error) {
+func (jh JobHandler) HandleStartPipelineJob(appName string, pipeline *pipelineJob.Definition, jobSpec *jobModels.JobParameters, environments map[string]bool) (*jobModels.JobSummary, error) {
+
+	appNamespace := k8sObjectUtils.GetAppNamespace(appName)
```

Comment:
This line can remain on its previous location (line 40)

Reply:
This has no functional advantage. Was placed at the top of the function to keep it more organized.